docs/en/dev/adapterdev.md

```diff
@@ -316,13 +318,37 @@ Short structure of package.json:
 ```
 - All fields are mandatory. devDependencies should be inside too to enable the grunt tasks.
 
+### Versioning
+
+It is required to use [Semantic Versioning](https://semver.org/) (SemVer) for your adapters.
+
+#### Syntax
+
+Version 1.2.3 =>
+
+| **Major** | **Minor** | **Patch** |
+|-----------|-----------|-----------|
+| 1         | 2         | 3         |
+
+#### Explanation
```

Comment:
maybe also tell something about 0.x and the specialities? Maybe also "when a developer should increase to 1.0"?

Reply:
I don't know that in detail, therefore I cannot provide this info at this time.

Reply:
basically it is:
For versions lower then 1.0 a minor update is breaking ... so for those release it changes in this regard ...

Reply:
Added accordingly. Please review. Not sure if this is really true for stable versions, but would be consistent with SemVer https://semver.org/#how-do-i-know-when-to-release-100